### DISCUSSION TITLE
Welcome to voicemail-express-amazon-connect Discussions!

### DISCUSSION THREAD

**Discussion by SteveEarl-AWS** [amazon-connect/voicemail-express-amazon-connect#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AbdulQayyumaq** [comment]
For VMXKVStoS3 Lambda, This error is being occured

{
    "timestamp": "2024-09-04T16:03:20Z",
    "level": "ERROR",
    "message": "An error occurred (InvalidArgumentException) when calling the ListFragments operation: End timestamp 1725465732000 is outside of the stream retention period.",
    "logger": "root",
    "requestId": "bedfe797-97f6-4716-895f-db9397445794"
}


And voicemail audio is not being delivered to s3 bucket.

**Comment by vmanzo1601** [comment]
Is there a way to increase the presigned recording URL to be greater than 7 days ?
